<launch>
	<arg name="external_camera" default="camera1" />
	<arg name="overhead_camera" default="camera2" />

	<node pkg="external_camera_localizer" type="transform_pointcloud" name="transform_external">
		<remap from="cloud" to="$(arg external_camera)/depth/points"/>
		<remap from="cloud_transformed" to="$(arg external_camera)/transformed"/>
		<param name="target_frame" type="string" value="world"/>
	</node>
	
	<!--node pkg="external_camera_localizer" type="transform_pointcloud" name="transform_overhead">
		<remap from="cloud" to="$(arg overhead_camera)/depth/points"/>
		<remap from="cloud_transformed" to="$(arg overhead_camera)/transformed"/>
		<param name="target_frame" type="string" value="$(arg overhead_camera)_full_link"/>
	</node-->

	<node pkg="modular_cloud_matcher" type="cloud_matcher_service" name="cloud_matcher_service" output="screen">
    <!--param name="config" value="$(find external_camera_localizer)/launch/matcher.yaml"/-->
  </node>
</launch>
